generators/entity/files.js

```diff
@@ -149,9 +149,9 @@ const serverFiles = {
             condition: generator => generator.gatlingTests,
             path: TEST_DIR,
             templates: [{
-                file: 'gatling/simulations/_EntityGatlingTest.scala',
+                file: 'gatling/user-files/simulations/_EntityGatlingTest.scala',
```

Comment:
To follow the new Gatling structure (http://gatling.io/docs/current/general/bundle_structure/)